src/main/java/de/zalando/aruha/nakadi/config/NakadiConfig.java

```diff
@@ -29,6 +33,11 @@
 
     public static final MetricRegistry METRIC_REGISTRY = new MetricRegistry();
     public static final HealthCheckRegistry HEALTH_CHECK_REGISTRY = new HealthCheckRegistry();
+    public static final String ZK_BROKERS_DEFAULT = "127.0.0.1:2181";
+    public static final String KAFKA_BROKERS_DEFAULT = "127.0.0.1:9092";
```

Comment:
I don't suggest to put default values to application.yml. I suggest to put ZK_BROKERS and KAFKA_BROKERS to application.yml. And that will already work as a default mechanism. (If no env. vars set - the values from application.yml will be used as defaults)